ZKSyncSDK/Classes/Provider/Provider.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Provider.swift
+//  ZKSyncSDK
+//
+//  Made with ❤️ by Matter Labs on 10/23/20
+//
+
+import Foundation
+import BigInt
+
+public protocol Provider {
```

Comment:
Correct me if I'm wrong, but I could not find a method that allows getting Ethereum transaction for a withdrawal. 

An example for js sdk:
https://github.com/matter-labs/zksync/blob/6c4a643acad0074958191bdd44cef4a7bd7a7206/sdk/zksync.js/src/provider.ts#L137

Reply:
`Provider` provides methods to submit transaction. Actual Withdraw transaction is generated in the `DefaultWallet` implementation and then submitted using Provider's submitTx method

Reply:
Please look at the js implementation and call the same JSON-RPC endpoint (`get_eth_tx_for_withdrawal`).

Withdraw = withdraw funds from zkSync. It has two transactions: the first one is on zkSync corresponding to user's funds being removed from his account, and the second one is on the Ethereum network that actually sends the funds to the user's Ethereum account.

This function lets you get the hash of the Ethereum transaction from the hash of the zkSync one.